pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -390,8 +390,10 @@ def newpkgcb(pkgId, name, arch):
             modulemd_results = None
             comps_downloader = None
             main_types = set()
+            checksums = {}
 
             for record in repomd.records:
+                checksums[record.type] = record.checksum_type.upper()
```

Comment:
I also wonder whether it makes sense to store checksum_type for every recortd.type. Maybe it would be enough just to find the `primary` because it would always be present in a repo and take its type.